edward/criticisms.py

```diff
@@ -5,18 +5,28 @@
 from edward.util import logit, get_session
 
 def evaluate(metrics, model, variational, data):
-    """
-    Evaluate fitted model using a set of metrics.
+    """Evaluate fitted model using a set of metrics.
 
     Parameters
     ----------
-    metric : list or str
+    metrics : list or str
         List of metrics or a single metric.
+    model : ed.Model
+        Probability model p(x, z)
+    variational : ed.Variational
+        Variational approximation to the posterior p(z | x)
+    data : ed.Data
+        Data to evaluate the model at
 
     Returns
     -------
     list or float
         A list of evaluations or a single evaluation.
+
+    Raises
+    ------
+    NotImplementedError
+        if an input metric does not match an implemented metric in Edward.
```

Comment:
Since we're following the docstring conventions more rigorously now, we should also enforce that the first character for each of these is capitalized; see, e.g., https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt